contextproject-ms3/src/main/java/contextproject/audio/EnergyLevelProcessor.java

```diff
@@ -53,30 +52,45 @@ public EnergyLevelProcessor(Attributes attributes) {
    * @throws LineUnavailableException
    *           Line unavailable exception.
    */
-  public void detect(Track track) throws EncoderException, LineUnavailableException {
+  public void detect(Track track, double start) throws EncoderException, LineUnavailableException {
     inputStream = Streamer.stream(track.getPath(), attributes);
     tarsosStream = new JVMAudioInputStream(inputStream);
 
     float msPerBeat = (float) (60.0f / track.getBpm());
-    float fourBarsInSeconds = msPerBeat * 16;
-    energyLevels = new ArrayList<Double>();
-    System.out.println(fourBarsInSeconds);
+    oneBarInSeconds = msPerBeat * 4;
+
+    System.out.println(oneBarInSeconds + " time per beat");
```

Comment:
Is a system.out needed, if yes please change it with the logger print, if not please remove.